prow/auth-presubmit.sh

```diff
@@ -40,7 +40,6 @@ else
 fi
 
 echo "=== Bazel Build ==="
-./bin/install-prereqs.sh
```

Comment:
Can we try to revert this line and try to rerun presubmit?

Reply:
it does this
```
go get -u github.com/alecthomas/gometalinter
gometalinter --install --vendored-linters
```
so shouldn't matter ? is there a way to run the test more verbose to see what happens during the 20s ?

Reply:
Seems this line is already removed in previous PR: https://github.com/istio/auth/pull/300/files

Did you forget to rebase?

Reply:
@sebastienvas , do you have any ideas about this presubmit error?

Reply:
I think someone (from auth) should look at what the test does/why it is failing

Reply:
> Seems this line is already removed [...]
@wattli not sure I understand your point... this PR includes #300 because otherwise the linters step doesn't pass - what do you mean ?

Reply:
@myidpt , you had a PR for integration test fix https://github.com/istio/auth/commit/fd08656b83caa6f96fd11bb13a266bad04b468e1, can you help to investigate this pre-submit error please? It seems to be integration test related.